nasa/cumulus-orca
# Feature/orca 317 website updates

## Summary of Changes

Updated ORCA inventory schema definition in the various file for the ORCA teams refined ORCA inventory research. This PR is for one of 3 tasks to complete the ticket.

Addresses [ORCA-317: Modify Orca catalog for new Cumulus schema](https://bugs.earthdata.nasa.gov/browse/ORCA-317)

## Changes

* Updated notes on research for `website/docs/developer/research/research-reconciliation.mdx`
* Updated schema definitions and layout for ADS file `website/static/files/ads_orca_schema_refined.xed`
* Updated refined SQL file for inventory `website/static/files/build_orca_inventory_refined.sql`

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

For the documentation, validated documentation build for website using:
- cd website
- npm install
- clear; npm run start --verbose -- --port 3001
- Validated output had no errors and pages displayed properly.

For the SQL file, verified that it ran properly in a Docker by performing the following.
- Create a postgres instance with Docker `docker run -d --rm -p 5432:5432 --name test-postgres -e POSTGRES_PASSWORD=mysecretpassword postgres:12`
- `cd` to the SQL folder in the git repository `cd website/static/files`
- determine your IP via `ipconfig getifaddr en0` to use for connecting to the database.
- Create a psql instance with Docker `docker run -it --rm -v `pwd`:/myfiles postgres:12 /bin/bash
- Log into postgres, install the sql and validate by performing the following in the interactive container.
   1.  log into the database via the command `psql --host <Your IP HERE> -U postgres postgres` password is the password set in the initlal postgres environment for the server container.
   2. Create the Database via the command `CREATE DATABASE orca;` at the psql prompt.
   3. Connect to the orca database and create the user and schema via the commands below at the psql prompt.
       1. `\c orca`
       2. `CREATE ROLE orca_dbo;`
       3. `CREATE SCHEMA orca AUTHORIZATION orca_dbo;`
   4. Run the SQL creation script via `\i /myfiles/build_orca_inventory_refined.sql;`
   5. Verify the tables were created
        ```
         orca=# \dt orca.*
            List of relations
            Schema |    Name     | Type  |  Owner
            --------+-------------+-------+----------
             orca   | collections | table | orca_dbo
             orca   | files       | table | orca_dbo
             orca   | granules    | table | orca_dbo
             orca   | providers   | table | orca_dbo
            (4 rows)  
       ```
## Checklist:

- [X] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [X] CHANGELOG.md
    - [X] Research documentation
    - [X] Manual tests (outlined above)
- [X] My code has passed security scanning
    - [X] Snyk
    - [X] git-secrets
